libs/api/mocks/src/domains/cms/store.ts

```diff
@@ -23,19 +23,19 @@ export const store = createStore(() => {
 
   // Create articles, groups and subgroups in every category.
   const articles = articleCategories.reduce<Article[]>((articles, category) => {
-    const groups = articleGroup.list(faker.datatype.number({ min: 2, max: 4 }))
+    const groups = articleGroup.list(faker.number.int({ min: 2, max: 4 }))
 
     for (const group of groups) {
       const subGroups =
-        faker.datatype.number(4) === 0
+        faker.number.int(4) === 0
           ? []
           : articleSubgroup.list(faker.datatype.number({ min: 2, max: 4 }))
 
       const groupArticles = article.list(subGroups.length * 2 + 2, {
         group,
         category,
         subgroup: () =>
-          subGroups.length > 0 ? faker.random.arrayElement(subGroups) : null,
+          subGroups.length > 0 ? faker.helpers.arrayElement(subGroups) : null,
```

Comment:
This suggestion was valid. Fixed in 04b53b2a0bdd865ee62efeb6bbdfc2bb22f850b8.

Reply:
This comment is correct but confusing, because I couldn't see that it refers to line 32.

I created a separate comment on that line:

- https://github.com/island-is/island.is/pull/17058/files/2d2f8e8dfbefff871ea810cc02daffffa68a4956#r1861278518